### PR TITLE
18. Recording/transcription broken on Safari

### PR DESCRIPTION
**Pull request by Andrew-Hird**
[Ticket](https://www.notion.so/palomagroup/18-Recording-transcription-broken-on-Safari-8ee58cbf5a6d4731bb548ac3cc5abcb1?pvs=4)

1. Add timeslice of `1000` to recorder start
2. Show spinner when MediaRecorder loading
3. Don't start clock until MediaRecorder loaded

### CODE REVIEW

**Review by rickyhopkins — CHANGES_REQUESTED** [review]
Definitely a step in the right direction but I think there are some edge cases that are not accounted for.

1. The stop recording button is clickable while microphone permissions are being requested, If it is clicked before permissions are granted then the page will throw an error.
2. This might be a little bit out of scope for this ticket but if permissions are blocked then we probably want to notify the user with some instructions on how to reset/allow permissions. Otherwise they will just be stuck on the loading screen.

**Comment by Andrew-Hird** [comment]
> Definitely a step in the right direction but I think there are some edge cases that are not accounted for.
> 
> 1. The stop recording button is clickable while microphone permissions are being requested, If it is clicked before permissions are granted then the page will throw an error.
> 2. This might be a little bit out of scope for this ticket but if permissions are blocked then we probably want to notify the user with some instructions on how to reset/allow permissions. Otherwise they will just be stuck on the loading screen.

@rickyhopkins have disabled the button until the MediaRecorder is ready, and have added a ticket to look at what to do if permissions are rejected.

File: components/Recorder.tsx [hunk 1]
```diff
@@ -167,23 +169,28 @@ const Recorder = ({ userId }: { userId: string }) => {
...
         <span
-          className={`block w-3 h-3 m-3 ${!isRunning && 'rounded-full'} bg-white`}
-        ></span>
-      </span>
-      <span className="text-2xl">
-        {isRunning ? 'Stop recording' : 'Record a note'}
-      </span>
-    </button>
-  );
+          className={`block rounded-full bg-${isRunning ? 'error-50 bg-orange-500' : 'orange-500 border-white'} mr-2 border-[1px]`}
```
Comment (rickyhopkins) [inline components/Recorder.tsx]: This will not work with tailwind JIT compiler
https://tailwindcss.com/docs/content-configuration#dynamic-class-names

**Review by rickyhopkins — CHANGES_REQUESTED** [review]
Much better, but the classes will not be working correctly

File: components/Recorder.tsx (hunk 1 above)
Comment (Andrew-Hird) [reply components/Recorder.tsx]: Done!

**Review by rickyhopkins — APPROVED** [review]

**Pull request merged**